### PR TITLE
feat: transcript parent widget component

### PR DESCRIPTION
**Pull request by KristinAoki**
This PR adds the parent widget component for video transcripts. Added snapshot tests to check that UI loads as expected.

JIRA Ticket: [TNL-9978)](https://2u-internal.atlassian.net/browse/TNL-9978)

### CODE REVIEW

File: src/editors/containers/ErrorAlerts/ErrorAlert.jsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Alert } from '@edx/paragon';
 import { Info } from '@edx/paragon/icons';
```
Comment (connorhaugh) [inline src/editors/containers/ErrorAlerts/ErrorAlert.jsx:5]: This move is straining the name `container` for the parent directory. I agree it needs to be a shared component, not inside text editor, but interested in what to call it.

File: src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+          </Form.Group>
+        ) : (
+          <>
+            <Alert variant="danger" icon={Info}>
+              Only SRT files can be uploaded. Please select a file ending in .srt to upload.
+            </Alert>
+            <FormattedMessage {...messages.addFirstTranscript} />
+          </>
+        )}
+        <Button iconBefore={FileUpload} onClick={() => { console.log('adding file'); }} variant="link">
```
Comment (connorhaugh) [inline src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:116]: You'll want this to be a paragon Dropzone I think.

Comment (connorhaugh) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:116]: Or at least be a button that opens up the file select object for now

Comment (connorhaugh) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:116]: Might be better to use an invisible input + button in this case like this: https://bobbyhadz.com/blog/react-open-file-input-on-button-click

File: src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx [hunk 3]
```diff
@@ -113,7 +115,8 @@ export const TranscriptWidget = ({
             <FormattedMessage {...messages.addFirstTranscript} />
           </>
         )}
-        <Button iconBefore={FileUpload} onClick={() => { console.log('adding file'); }} variant="link">
+        <FileInput fileInput={fileInput} acceptedFiles=".srt" />
```
Comment (connorhaugh) [inline src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:118]: Did you do any CSS to hide this fileinput, or does it show up on the UI?

Comment (connorhaugh) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:118]: Just the one quick question

Comment (connorhaugh) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:118]: otherwise looks good

Comment (KristinAoki) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:118]: No CSS and it doesn't appear in the UI. The button controls the ref that provides the functionality of an `input` element. I am following the same structure that is used in the `SelectImageModal`

Comment (connorhaugh) [reply src/editors/containers/VideoEditor/components/VideoSettingsModal/components/TranscriptWidget/index.jsx:118]: Ok Awesome! I will give the thumb and we can merge this in and i will rebase off it.

**Review by connorhaugh — APPROVED** [review]

**Pull request merged**